nuitka/tools/release/Documentation.py

```diff
@@ -113,33 +110,29 @@ def checkRstLint(document):
     print("OK.")
 
 
+@which("help2man")
```

Comment:
So is all this is doing to give a nicer error message of the first argument cannot be resolved? Not sure if that's worth it, a wrapper in `nuitka.utils.Execution` could use `try`/`except` on the process call error and output something nicer.

Reply:
I think this code is mostly used by packagers this - a clear error is quite a help. If this code was used by others codes, than try except is better.
You decide. If you want I can remove this decorator completely.

Reply:
I think, having a wrapper of `subprocess.checked_call` that catches the error in case of program not found, would be good enough. 

```
try: subprocess.checked_call(args)
except subprocess.SomeExceptionWeNeedToLoopUp as e:
   if e.something:
       sys.exit("Program args[0] not found!")
   else:
      raise
```

```
if os.path.sep not in args[0]:
    args=list(args)
    prog = getExecutablePath(args[0))
    if prog is None:
       sys.exit("Program not found!")
```

I think the later is better for extension, because I have places, where e.g. some tools are called, with `python -m prog` in preference to `prog` and I could merge that there. Also I like checks better than exception catches, even if Python is good at those.

I agree the Python call stack is barely useful to normal people, and having a proper system exit that explains an installation problem would be very nice. There is something about these tracebacks that turns off people's ability to continue reading. :-)

Reply:
Honestly, I don't know where you would put this wrapper for `check_call` ... I see there is already
a `check_output` for compat with Python 2.6. Would like to me to add those changes there?
Do you really still want to support Python2.6? It's been dead for quite a while...

Reply:
Just to respond to this, Python 2.6 is used on enterprise systems still. For Nuitka it's not a big deal, because 2.7 is really only a 2.6 with the then best Python3 features backported to ease the migration, e.g. dict and set contractions, but nothing really important.

To Nuitka, Python is translated into a simpler language right after ast parsing, and the non-availability of higher language constructs doesn't make a problem there.

In terms of support effort, it's minimal, but test coverage is nice to have. I have not spent time on Python 2.6 for ages. Should I remove the support, very little code could go away.

As for making the changes there, you can go ahead. I am satisfied with your PR as it is though, this maybe is something I should do, as it affects many tools you don't care about. Let me know, if you want to persue this further. Otherwise I can pick up these ideas and use them.

Reply:
Thanks for the reply. I think my latest push has all your suggestions. It's just a bit hard to really know how you would implement this. I would not mind if you took a look in the latest chnages and say if you are satisfied or not.

Reply:
Thanks for the good job.